### ISSUE TITLE
Societies migration throws error

### ISSUE THREAD

**Issue by thaliawww-service** [thaliawww/concrexit#699]
In GitLab by @joren485 on Sep 12, 2018, 20:36

When running the of migrations for ce7f1c89342a8914eb2076bb41e4aaf89ebc24ae on beta with production data the following error is thrown:

```
Running migrations:
  Applying activemembers.0027_auto_20180901_1545... OK
  Applying activemembers.0028_create_tmp_board_model... OK
  Applying activemembers.0029_save_board_pk_in_tmp... OK
  Applying activemembers.0030_delete_original_board_model... OK
  Applying activemembers.0031_rename_base_model... OK
  Applying activemembers.0032_create_submodels... OK
  Applying activemembers.0033_fill_submodel_schemas...Traceback (most recent call last):
  File "/usr/local/lib/python3.6/site-packages/django/db/backends/utils.py", line 85, in _execute
    return self.cursor.execute(sql, params)
psycopg2.ProgrammingError: column "evenementencommissie" does not exist
LINE 1: ... (membergroup_ptr_id, wiki_namespace) VALUES (31, evenemente...
                                                             ^


The above exception was the direct cause of the following exception:

Traceback (most recent call last):
  File "./manage.py", line 19, in <module>
    execute_from_command_line(sys.argv)
  File "/usr/local/lib/python3.6/site-packages/django/core/management/__init__.py", line 371, in execute_from_command_line
    utility.execute()
  File "/usr/local/lib/python3.6/site-packages/django/core/management/__init__.py", line 365, in execute
    self.fetch_command(subcommand).run_from_argv(self.argv)
  File "/usr/local/lib/python3.6/site-packages/django/core/management/base.py", line 288, in run_from_argv
    self.execute(*args, **cmd_options)
  File "/usr/local/lib/python3.6/site-packages/django/core/management/base.py", line 335, in execute
    output = self.handle(*args, **options)
  File "/usr/local/lib/python3.6/site-packages/django/core/management/commands/migrate.py", line 200, in handle
    fake_initial=fake_initial,
  File "/usr/local/lib/python3.6/site-packages/django/db/migrations/executor.py", line 117, in migrate
    state = self._migrate_all_forwards(state, plan, full_plan, fake=fake, fake_initial=fake_initial)
  File "/usr/local/lib/python3.6/site-packages/django/db/migrations/executor.py", line 147, in _migrate_all_forwards
    state = self.apply_migration(state, migration, fake=fake, fake_initial=fake_initial)
  File "/usr/local/lib/python3.6/site-packages/django/db/migrations/executor.py", line 244, in apply_migration
    state = migration.apply(state, schema_editor)
  File "/usr/local/lib/python3.6/site-packages/django/db/migrations/migration.py", line 122, in apply
    operation.database_forwards(self.app_label, schema_editor, old_state, project_state)
  File "/usr/local/lib/python3.6/site-packages/django/db/migrations/operations/special.py", line 190, in database_forwards
    self.code(from_state.apps, schema_editor)
  File "/usr/src/app/website/activemembers/migrations/0033_fill_submodel_schemas.py", line 22, in migrate_to
    .format(group.pk, group.wiki_namespace_old if group.wiki_namespace_old else 'NULL'))
  File "/usr/local/lib/python3.6/site-packages/django/db/backends/base/schema.py", line 133, in execute
    cursor.execute(sql, params)
  File "/usr/local/lib/python3.6/site-packages/django/db/backends/utils.py", line 100, in execute
    return super().execute(sql, params)
  File "/usr/local/lib/python3.6/site-packages/raven/contrib/django/client.py", line 127, in execute
    return real_execute(self, sql, params)
  File "/usr/local/lib/python3.6/site-packages/django/db/backends/utils.py", line 68, in execute
    return self._execute_with_wrappers(sql, params, many=False, executor=self._execute)
  File "/usr/local/lib/python3.6/site-packages/django/db/backends/utils.py", line 77, in _execute_with_wrappers
    return executor(sql, params, many, context)
  File "/usr/local/lib/python3.6/site-packages/django/db/backends/utils.py", line 85, in _execute
    return self.cursor.execute(sql, params)
  File "/usr/local/lib/python3.6/site-packages/django/db/utils.py", line 89, in __exit__
    raise dj_exc_value.with_traceback(traceback) from exc_value
  File "/usr/local/lib/python3.6/site-packages/django/db/backends/utils.py", line 85, in _execute
    return self.cursor.execute(sql, params)
django.db.utils.ProgrammingError: column "evenementencommissie" does not exist
LINE 1: ... (membergroup_ptr_id, wiki_namespace) VALUES (31, evenemente...
```

**Comment by thaliawww-service** [comment]
In GitLab by @se-bastiaan on Sep 13, 2018, 14:40

mentioned in commit 216d7bf7b1c41fc99b515b1264dfe71e2275814a

**Comment by thaliawww-service** [comment]
In GitLab by @se-bastiaan on Sep 13, 2018, 14:44

mentioned in commit 73fa5984f2862ad60d5f20a0eec0de6fc52dc9c2

**Comment by thaliawww-service** [comment]
In GitLab by @joren485 on Sep 13, 2018, 15:16

closed via merge request !940

**Comment by thaliawww-service** [comment]
In GitLab by @se-bastiaan on Sep 13, 2018, 15:16

closed via commit 73fa5984f2862ad60d5f20a0eec0de6fc52dc9c2

**Comment by thaliawww-service** [comment]
In GitLab by @joren485 on Sep 13, 2018, 15:16

mentioned in commit 04c619e42cf96c8a5c8972cbf7de0e20c65d5cea
